packages/react/src/QuestionnaireBuilder/QuestionnaireBuilder.tsx

```diff
@@ -204,6 +230,15 @@ function ItemBuilder<T extends Questionnaire | QuestionnaireItem>(props: ItemBui
     });
   }
 
+  function toggleItemOrder(itemToMove: QuestionnaireItem, delta: number): void {
```

Comment:
It's not really a "toggle"... Maybe `moveItem`?

In this case, I think your code will end up cleaner if you accept the `itemIndex` as well rather than `itemToMove`.  When passing in the `QuestionnaireItem`, you have to do an `Array.find()` which is O(n) lookup.  You already have the item index on lines 295-296, so might as well use it directly.